.github/workflows/amorphie.core.cache.yml

```
@@ -0,0 +1,27 @@
+    branches: [ main ]
+    paths:
+      - 'amorphie.core.security/**'
+env:
+  APP_NAME: amorphiecore.security
+
+jobs:
+  ci-cd-worker-app:
+    if: "!startsWith(github.head_ref, 'feature/')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)